Library/Formula/leiningen.rb

```diff
@@ -53,5 +51,4 @@ def caveats; <<-EOS.undent
     EOS
     system "#{bin}/lein", "test"
```

Comment:
@mikemcquaid Fixed, thanks. Would there be any way to get an email notification when a PR build fails?